### DISCUSSION TITLE
blog/2019-10-14-individuals-are-not-small-groups-ii-the-ecological-fallacy/

### DISCUSSION THREAD

**Discussion by giscus** [ASKurz/blogdown5#24] [Comments]
# blog/2019-10-14-individuals-are-not-small-groups-ii-the-ecological-fallacy/

When people conclude results from group-level data will tell you about individual-level processes, they commit the *ecological fallacy*. This is true even of the individuals whose data contributed to those group-level results. This phenomenon can seem odd and counterintuitive. Keep reading to improve your intuition.

https://solomonkurz.netlify.app/blog/2019-10-14-individuals-are-not-small-groups-ii-the-ecological-fallacy/

**Comment by darylchc** [comment]
Solomon, I just wanna say a big thanks to this post. It has really helped me understand ecological fallacy, especially with the typing example. ~ Daryl Chow
